db/seeds.rb

```diff
@@ -5,3 +5,8 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+users = [
+    { email: 'user@gmail.com', password: '123456' }
```

Comment:
looks not so full as it could be, try to create really good seeds, which could be used by others.